modules/ossm-migrating-read-me-support-for-istioctl.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/about/ossm-migrating-assembly.adoc
+
+//Start of an overall Migrating section.
+//Section is most likely to be reworked/restructured with OSSM 2 to OSSM 3 migration guides for GA. Unknown how many migration guides there are at this time (11/11/2024). It would be beneficial to be able to link from differences to the relevent migration guide so that users A) understand the change, esp significant changes like new operator, installing tracing and Kiali separately, gateways, etc.
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-migrating-read-me-support-for-istioctl_{context}"]
+= Support for Istioctl
+
+{SMProduct} 1 and 2 did not include support for Istioctl, the command line utility for the Istio project that includes many diagnostic and debugging utilties. {SMProduct} 3 introduces support for Istioctl for select commands.
+
+.Supported Istioctl commands
+[cols="1,1"]
+|===
```

Comment:
Note that I think this table should be moved to the future page on istioctl when it is available (covered by https://issues.redhat.com/browse/OSSM-4028), because this OSSM 2 vs 3 page will likely be removed at some point (long after OSSM 2 is EOLed) and we will want to maintain a list of supported istioctl commands for users who are not concerned with OSSM 2.

Reply:
Discussed with Gwynne, and we'll leave this here for now until the istioctl page is published.